sunpy/net/tests/test_fido.py

```diff
@@ -489,3 +489,10 @@ def test_slice_jsoc():
 
     with pytest.warns(SunpyUserWarning) as record:
         Fido.fetch(res[0, 0:1])
+
+
+@pytest.mark.remote_data
+def test_response_block_properties():
+    res = Fido.search(a.Time('2012/3/4', '2012/3/6'), a.Instrument('aia'), a.Wavelength(171 * u.angstrom),
+                      a.vso.Sample(10 * u.minute))
+    assert len(res) == 0
```

Comment:
How does this test the `response_block_properties` function?

Reply:
sorry I forgot to use the function.